lib/std/zig/Zir.zig

```diff
@@ -2035,10 +2042,22 @@ pub const Inst = struct {
         /// Implement builtin `@errorFromInt`.
         /// `operand` is payload index to `UnNode`.
         error_from_int,
-        /// Implement builtin `@Type`.
+        /// Implement builtin `@Enum`.
+        /// `operand` is payload index to `Reify`.
+        /// `small` contains `NameStrategy`.
+        enum_reify,
+        /// Implement builtin `@Pointer`.
+        /// `operand` is payload index to `Reify`.
+        /// `small` contains `NameStrategy`.
+        pointer_reify,
```

Comment:
Because pointers are not container types, they don't need an absolute AST node or line number attached. I would therefore expect this to use `UnNode` instead of `Reify`.